tests/testTriggers.go

```diff
@@ -5898,12 +5898,13 @@ func TriggerAsyncDRVolumeOnly(contexts *[]*scheduler.Context, recordChan *chan *
 		if get_mig_err != nil {
 			UpdateOutcome(event, fmt.Errorf("failed to get migration: %s in namespace %s. Error: [%v]", mig.Name, mig.Namespace, get_mig_err))
 		}
+		volumesMigrated := resp.Status.Summary.NumberOfMigratedVolumes
 		resourcesMigrated := resp.Status.Summary.NumberOfMigratedResources
-		if resourcesMigrated != 0 {
-			UpdateOutcome(event, fmt.Errorf("resources should not migrate in volumeonlymigration case, numberOfmigratedresources should %d, getting %d",
-				0, resourcesMigrated))
+		expectedresourcesMigrated := 2 * volumesMigrated
+		if resourcesMigrated != expectedresourcesMigrated {
+			UpdateOutcome(event, fmt.Errorf("resources migrated for should be %d, getting %d", expectedresourcesMigrated, resourcesMigrated))
```

Comment:
Can we change it to `fmt.Errorf("Number of resources migrated should be %d, got %d", expectedresourcesMigrated, resourcesMigrated)`, so it matched the else print

Reply:
Thanks, Done!